### ISSUE TITLE
Error in packer_compiled: Failed to load parser: uv_dlsym: dlsym(0x30884af80, tree_sitter_Markdown): symbol not found

### ISSUE THREAD

**Issue by MatthewCushing** [0x100101/lab.nvim#13]
I'm running into an issue where I receive the error:

```
Error in packer_compiled: Failed to load parser: uv_dlsym: dlsym(0x30884af80, tree_sitter_Markdown): symbol not found
```

Also, is there something different that needs to be done to support `typescriptreact` with `.tsx` files?  It would be great to get support for this filetype.

**Comment by 0x100101** [comment]
@MatthewCushing nice catch with the `tsx` issue. It's supposed to work but a typo is preventing it. See here: https://github.com/0x100101/lab.nvim/blob/main/lua/lab/code_runner.lua#L34 `script` is missing its `s`. I'll be releasing a fix soon.

As for the other error you encountered, I'm almost certain it's unrelated to lab.nvim?

**Comment by MatthewCushing** [comment]
> @MatthewCushing nice catch with the `tsx` issue. It's supposed to work but a typo is preventing it. See here: https://github.com/0x100101/lab.nvim/blob/main/lua/lab/code_runner.lua#L34 `script` is missing its `s`. I'll be releasing a fix soon.
> 
> 
> 
> As for the other error you encountered, I'm almost certain it's unrelated to lab.nvim?

I was hoping the tsx problem would be something simple like that. Yay. As for the error I'm encountering, I'm not so sure. I only encounter it with your plugin. I looked through your code a bit and saw that you are using `tree_sitter_Markdown` I believe. Think I remember seeing that you are using Markdown in the side window? I made sure to download markdown and inline markdown from Treesitter, but it had no effect. I might have seen markdown in your JS files and wasn't sure if trouble was coming from there or if I just needed to install something 3rd party. I'll try running this on a more barebones nvim install and see if I still receive the same issue.

Thanks!

**Comment by al-ce** [comment]
Hi all, I was running into this problem after switching my config from a packer based one to a [lazy.nvim](https://github.com/folke/lazy.nvim) one on 2023-Jan-09. Long story short, I managed to get lab running again after simply changing `Markdown` to `markdown` in line 37 of `panel.lua`.

```lua
api.nvim_buf_set_option(self.buf_handle, "filetype", "markdown")
```

As I said, this is the change I have to make on a fresh config that just installed Treesitter yesterday. I don't have to make that change to let lab work on my old config, even if I flip back and forth (with the caveat that I change back to the old files in `~/.local/share/nvim` as well as the old `init.lua`). Moreover, that old config runs `:TSUpdate` on every Packer Sync, but I don't know what that really means as far as previously installed languages.

This may be unrelated, but I also had to manually run the `npm ci` command instead of trying to get `lazy` to do it for me, but that didn't fix the `m/M` markdown issue anyway. Not going to make a PR for this because I don't want to assume this is a universally applicable fix. @MatthewCushing let me know if this works for you, just out of curiosity and also in case this breaks for me again in the future, might give me more to work with.

@0x100101 thanks so much for lab, it's such an essential part of my workflow. Best wishes and happy new year to all

**Comment by 0x100101** [comment]
Hey @al-ce thanks for the kind words and detailed info.

@MatthewCushing I'm also interested to know if this solves the problem for you?

I'll spend some time with this and try to repro the problem. But even if I can't, if the lowercase `markdown` seems to work universally I'll switch it over.

**Comment by MatthewCushing** [comment]
> Hey @al-ce thanks for the kind words and detailed info.
> 
> @MatthewCushing I'm also interested to know if this solves the problem for you?
> 
> I'll spend some time with this and try to repro the problem. But even if I can't, if the lowercase `markdown` seems to work universally I'll switch it over.

@0x100101 I found that I received that error or similar errors by trying to `require('lab')` in the config or setup function inside of packer.  Once I took the Lab setup function outside of packer, I stopped receiving the error everywhere.  I'm just importing the file with the setup function inside of `init.lua`.  However, I still receive the error in `tsx` files.  I believe that is just a side effect of the filetype not being supported.  Otherwise, I've gotten it to work.  Seems to be one of those odd errors that shows up and points you in a direction that has nothing to do with where the problem actually resides.

**EDIT:** I've also found a much more worrisome bug where some plugins (NvimTree for example) do not work when opening files with filetypes that are not supported by `Lab.nvim`.  I also confirmed that I receive that markdown error on all filetypes that are not supported by the plugin.
